### PR TITLE
scrape.js: Find more user GitHub profiles

### PR DESCRIPTION
**Pull request by andrewda**
This finds more user profiles by getting their created issues and
prs in addition to their commits.

Extension of #62

### CODE REVIEW

File: lib/scrape.js [hunk 1]
```diff
@@ -43,32 +44,57 @@ async function fetchLeaders(id) {
...
-  let match = commitPattern.exec(body)
+  const commitsBody = await commitsRes.text()
+  const issuesBody = await issuesRes.text()
+  const prsBody = await prsRes.text()
+  const reviewsBody = await reviewsRes.text()
+
+  const repos = [...findMatches(commitsBody, commitPattern),
+                 ...findMatches(issuesBody, otherPattern),
+                 ...findMatches(prsBody, otherPattern),
+                 ...findMatches(reviewsBody, otherPattern)]
```
Comment (blazeu) [inline lib/scrape.js]: Is it possible to use `Promise.all` here?

Comment (blazeu) [reply lib/scrape.js]: ^ So the request can be done in parallel.

File: lib/scrape.js [hunk 2]
```diff
@@ -43,32 +44,57 @@ async function fetchLeaders(id) {
...
+    `${GH_USER_BASE}/${user}/created_commits?from=${from}&to=${to}`
+  )
+  const issuesRes = await fetch(
+    `${GH_USER_BASE}/${user}/created_issues?from=${from}&to=${to}`
+  )
+  const prsRes = await fetch(
+    `${GH_USER_BASE}/${user}/created_pull_requests?from=${from}&to=${to}`
+  )
+  const reviewsRes = await fetch(
+    `${GH_USER_BASE}/${user}/created_pull_request_reviews?from=${from}&to=${to}`
```
Comment (blazeu) [inline lib/scrape.js]: ^ So the request can be done in parallel.

**Review by jayvdb — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/74/commits/6964eae0b8c97f39a8bb23ba28b61b8b5d18d23e

**Pull request merged**